NEWS.txt

```diff
@@ -6,10 +6,10 @@ Note:
   - The .mm2 file format changed. Old files can be read, but files saved
     with this version can't be used with MMapper 2.3.x or older.
 Changes:
-  - Added extra door flags (teoli)
   - Added support for classes of info marks, with different colors and font decorations (teoli)
   - Added ability to rotate info marks (teoli)
   - Fixed wall color when a special exit was present (was white instead of black) (teoli)
+  - Added ability to store direction of river flows and display it on the map (telling)
```

Comment:
Oups, it should be 'teoli'. And I still need to rebase properly (I noticed that it has commits it shouldn't have)